### PR TITLE
Script improvements

### PR DESCRIPTION
**Pull request by Margen67**
shebang:
 Move parameters from set.
 Add newline after for consistency.
curl:
 Add -f;
  Prevents output on error.
 Use -O where applicable;
  -o with the same filename as the URL is pointless.
 Add -S after -s:
  Makes curl still error despite being silent.
build-symbols:
 Add zip fallback.
get-discord-rpc:
 Move git clone parameters before repository for readability.
 Use --depth 1 to speed up cloning.
install-nsis:
 Use ProgramData environment variable.
build-appimage-docker.sh:
 Add -e to docker run.
 Run script directly.
build-appimage.sh:
 Make if more consistent.
Remove unused Travis scripts.
setenv:
 unset instead of blanking variables.
vstool.cmd:
 Use ProgramFiles(x86) environment variable.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Any reasons for these changes?

**Comment by Margen67** [comment]
> 
> 
> Any reasons for these changes?

Look at the commit history.

**Comment by duncanspumpkin** [comment]
Okay I've copied that into the pr comment so it's not missed

File: scripts/build-symbols [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
-
 echo -e "\033[0;36mCreating symbols archive for OpenRCT2...\033[0m"
 if [[ -f $destination ]]; then
     rm $destination
 fi
-7z a $destination openrct2.exe openrct2.com openrct2-win.pdb
+files=(openrct2.exe openrct2.com openrct2-win.pdb)
+case "$archiver" in
+    7z)
+        7z a $destination ${files[@]}
```
Comment (tupaschoal) [inline scripts/build-symbols:26]: Can you make archiver be `7z a`? Then we wouldn't need this `case` here

**Review by tupaschoal — APPROVED** [review]
This looks fine to me. @janisozaur and @IntelOrca any thoughts?

**Comment by IntelOrca** [comment]
@tupaschoal fine with me

**Comment by janisozaur** [comment]
This broke our PPA: https://launchpad.net/~openrct2/+archive/ubuntu/nightly/+build/21393926/+files/buildlog_ubuntu-hirsute-arm64.openrct2_0.3.3-develop-0~2447-202104122212-g1b64fc871~ubuntu21.04.1_BUILDING.txt.gz

**Comment by tupaschoal** [comment]
I don't really understand what broke

**Comment by YJSoft** [comment]
@janisozaur 
```
Downloading /<<PKGBUILDDIR>>/debian/openrct2/usr/share/openrct2/sequence/
CMake Error at /<<PKGBUILDDIR>>/cmake/download.cmake:23 (file):
  file DOWNLOAD HASH mismatch

    for file: [/<<PKGBUILDDIR>>/debian/openrct2/usr/share/openrct2/sequence//title-sequences.zip]
      expected hash: [304d13a126c15bf2c86ff13b81a2f2cc1856ac8d]
        actual hash: [da39a3ee5e6b4b0d3255bfef95601890afd80709]
             status: [6;"Couldn't resolve host name"]

Call Stack (most recent call first):
  cmake_install.cmake:52 (download_openrct2_zip)


make[1]: *** [Makefile:96: install] Error 1
make[1]: Leaving directory '/<<PKGBUILDDIR>>/obj-aarch64-linux-gnu'
dh_auto_install: error: cd obj-aarch64-linux-gnu && make -j1 install DESTDIR=/<<BUILDDIR>>/openrct2-0.3.3-develop-0\~2447-202104122212/debian/openrct2 AM_UPDATE_INFO_DIR=no returned exit code 2
make: *** [debian/rules:6: binary-arch] Error 25
dpkg-buildpackage: error: fakeroot debian/rules binary-arch subprocess returned exit status 2
```

CMake just failed to download title-sequences.zip(which is not related to this PR)

**Comment by janisozaur** [comment]
It's not that it "couldn't download" - it's expected, as the machines have no internet access. They shouldn't be even trying to download it in the first place, as the expected data should be there already.

I may have rushed my opinion, as I saw cmake changes and shortly after - this breakage. I still find it the most suspect one, but haven't had chance to prove it guilty just yet.

**Comment by janisozaur** [comment]
It's not this PR that broke PPA, but #14313

**Comment by tupaschoal** [comment]
Thanks for double checking!

**Pull request merged**